tendrl/node_agent/manager/pull_hardware_inventory.py

```diff
@@ -1,4 +1,4 @@
-from command import Command
+from tendrl.commons.utils import cmd_utils
```

Comment:
Unless there is a patch to change the current commons. This should be something like:
from tendrl.common.utils import command